### ISSUE TITLE
server name: error fetching (failed to parse

### ISSUE THREAD

**Issue by grilor** [TheImaginear/dynamiX#1]
windows 11  pro
unzipped the files into a folder
ran the dynamiXMain.exe
filled in all the information required
once clicking the save button i get this error
Server Name: error fetching (failed to parse: http://xxx.xxx.xxx.xxx:32400)

I have tried local ip 192.168.xxx.xxx
i have tried external ip xxx.xxx.xxx.xxx (x are to hide my ip)

i deleted the config file and did the setup 3 more time all with the same issue

EDIT:
fixed the 1st part. i had some how put a space between the last ip number and the : 

but now im getting this error and this is over my head now:

An error occurred: ('Connection aborted.', ConnectionResetError(10054, 'An existing connection was forcibly closed by the remote host', None, 10054, None))

i have looked into this and now im stuck

**Comment by TheImaginear** [comment]
Are you sure you entered the plex token correctly? That seems to be showing that the plex server is rejecting the connection. Make sure you have entered the plex token correctly (https://support.plex.tv/articles/204059436-finding-an-authentication-token-x-plex-token/), be sure that your server is available in your browser at the url you entered, and try this in your command prompt to ensure that you are able to establish a connection with the token and url: curl -X GET http://<server_ip>:32400/?X-Plex-Token=<your_token>

I would recommend removing the config file before updating your token to ensure its updated correctly.

**Comment by TheImaginear** [comment]
I'm certain it has something with how it interacts with the vhost. Ill try to address this in a future build.

**Comment by grilor** [comment]
Sounds good, thanks!

**Comment by TheImaginear** [comment]
In plex settings. Network, Secure Connections. Set to Preferred. I receive your exact error when that is set to "Required"

**Comment by grilor** [comment]
Ok, I'll check it out.

**Comment by TheImaginear** [comment]
Were you able to get it working? Ill be closing this issue on 1/10 if there is no response.

**Comment by grilor** [comment]
No, not yet. But it's been a rough xmas and new year here. Had a death in
the family and daughter might need knee surgery. but close it and if I need
to I can repost a new one if needed. Sorry for taking up your time.

**Comment by TheImaginear** [comment]
No problem at all, just wanted to clean up the issues that I could! I'll restrict this issue to the two of us for now. Hope things are better for you soon, happy new year.
